src/PointwiseFunctions/MathFunctions/RegisterDerivedWithCharm.hpp

```diff
@@ -0,0 +1,7 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+void register_derived_with_charm();
+  // namespace domain::CoordinateMaps::MathFunctions
```

Comment:
You have the docs for a namespace, but no namspace. Please wrap this function in a namespace (probably `MathFunctions::`). Also add `_1d` to the name if you're only going to add 1D math functions.